README.md

```diff
@@ -1590,6 +1592,7 @@ Update Time, five active automations, webhooks.
    * [codesandbox.io](https://codesandbox.io/) — Online Playground for React, Vue, Angular, Preact, and more.
    * [Components.studio](https://webcomponents.dev/) - Code components in isolation, visualize them in stories, test them, and publish them on npm.
    * [Eclipse Che](https://www.eclipse.org/che/) - Web-based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community-driven. An online instance hosted by Red Hat is available at [workspaces.openshift.com](https://workspaces.openshift.com/).
+   * [Emacs](https://www.gnu.org/software/emacs/) -Emacs
```

Comment:
I dont think we'll be adding this one, its in every distro, people know it.  We list here more out of the box things like projects thats not so ingrained, I am certainly not looking to list ever single open source project here.

Same for vi of course